web/src/composebox_typeahead.js

```diff
@@ -234,7 +234,6 @@ function handle_keydown(e) {
                         compose_validate.warn_for_text_overflow_when_tries_to_send() &&
                         !$("#compose-send-button").prop("disabled")
                     ) {
-                        $("#compose-send-button").prop("disabled", true);
```

Comment:
Looks like the only check of `$("#compose-send-button").prop("disabled")` is here, so there shouldn't be any side effects of not disabling the button.